src/app/MainMenu.js

```diff
@@ -97,7 +98,7 @@ export const MainMenu = (activePage) => {
 
   //create search
   const search = createElementWithClasses('form', 'search')
-  const searchInput = createElementWithClasses('input');
+  const searchInput = createElementWithClasses('input',);
   const searchBtn = createElementWithInnerText('button', 'Search')
```

Comment:
Your have to add class or ids to your elements in searching section and then refer to them by it, because you override buttons and inputs from my feature (see the video i sent on discord in 'radosław-bajor-projekt-2'

Reply:
And if anynone else also have forms in theirs features, they won't work

Reply:
const buttonForSearch = document.querySelector('form button'); <-- the query selector took my button instead of yours